manifests/cf-manifest/operations.d/249-bump-cflinuxfs3-release.yml

```diff
@@ -0,0 +1,8 @@
+---
+- type: replace
+  path: /releases/name=cflinuxfs3
+  value:
+    name: cflinuxfs3
+    version: "0.46.0"
+    url: "https://bosh.io/d/github.com/cloudfoundry/cflinuxfs3-release?v=0.46.0"
+    sha1: "0dae6d614dec4dd5db40ec01e62c2e8c5470c2d9"
```

Comment:
We already have a `229-cf-update_cflinuxfs_releases.yml` opsfile. Can we update the version there instead of creating a new opsfile.